src/templates/settings.html

```diff
@@ -63,7 +63,7 @@
     </div>
   </section>
 
-  <section class="main full hide-on-desktop account-settings-links">
```

Comment:
Here there is reviewer tools button. So i thought if a reviewer wants to use these in their tablet,  they should have these buttons. Rather we can hide the `footer-extras` links as we have the privacy,  terms button available here along with get involved button & reviewer button. What do you think? :)